relm4-macros/src/visitors.rs

```diff
@@ -98,26 +110,128 @@ impl VisitMut for ComponentVisitor {
                     }
                     remove = true;
                 }
-                "pre_view" => {
-                    let existing = self.pre_view.replace(func.clone());
-                    if existing.is_some() {
-                        self.errors
-                            .push(syn::Error::new_spanned(func, "duplicate pre_view function"));
+            }
+            _ => (),
+        }
+
+        if remove {
+            *item = syn::ImplItem::Verbatim(quote! {});
```

Comment:
Since you've already defined a function for that, this should be `null_item()` too.

Reply:
Fixed.